guide/students/student_management_service.bal

```diff
@@ -0,0 +1,322 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+import ballerina/mysql;
+import ballerina/observe;
+import ballerina/runtime;
+
+// Type Student is created to store details of a student.
+type Student record {
+    int id;
+    int age;
+    string name;
+    int mobNo;
+    string address;
+};
+
+// End point for marks details client.
+endpoint http:Client marksService {
+    url: " http://localhost:9191"
+};
+
+// Endpoint for mysql client.
+endpoint mysql:Client testDB {
+    host: "localhost",
+    port: 3306,
+    name: "testdb",
+    username: "root",
+    password: "",
+    poolOptions: { maximumPoolSize: 5 },
+    dbOptions: { useSSL: false }
+};
+
+// This service listener.
+endpoint http:Listener listener1 {
```

Comment:
Shall we use meaningful names for endpoints, variables etc.?

Reply:
Noted.  I will do the necessary changes.